### PR TITLE
Send Subscibe CRUD request from client to parodus

### PR DESCRIPTION
**Pull request by selvamKrish**

### CODE REVIEW

File: src/upstream.c [hunk 1]
```diff
@@ -309,6 +336,54 @@ void *processUpstreamMessage()
                         ParodusInfo(" Received upstream data with MsgType: %d dest: '%s' transaction_uuid: %s\n", 
                                       msgType, msg->u.req.dest, msg->u.req.transaction_uuid );
                     } else {
+                    	
+                    	if ( WRP_MSG_TYPE__CREATE == msgType && msg->u.crud.source != NULL && strstr(msg->u.crud.source,"subscribe")!= NULL) {
```
Comment (shilpa24balaji) [inline src/upstream.c]: Add "/parodus/subscribe" string compare

File: src/upstream.c [hunk 2]
```diff
@@ -309,6 +336,54 @@ void *processUpstreamMessage()
...
+                    		ParodusInfo("source %s\n",msg->u.crud.source);
+                    		parStrncpy(sourceVal,msg->u.crud.source, sizeof(sourceVal));
+                    		strtok(sourceVal , "/");
+                        	parStrncpy(dest,strtok(NULL , "/"), sizeof(dest));
```
Comment (shilpa24balaji) [inline src/upstream.c]: Add comments explaining why '/' is stripped twice

**Comment by shilpa24balaji** [comment]
As discussed with Wes, we should do the handling in CRUD queue, better to be consistent

**Comment by shilpa24balaji** [comment]
As discussed with @schmidtw we should do the handling from CRUD queue, better to be consistent.

File: src/upstream.c [hunk 3]
```diff
@@ -178,6 +202,9 @@ void *processUpstreamMessage()
     reg_list_item_t *temp = NULL;
     int matchFlag = 0;
     int status = -1;
+    char sourceVal[32] = {'\0'};
```
Comment (shilpa24balaji) [inline src/upstream.c]: I array required here, cant we use pointer. 32 looks strict length restriction

**Comment by selvamKrish** [comment]
Given new PR https://github.com/schmidtw/parodus/pull/29

**Pull request closed without merging**